packages/plugins/rubygem-hammer_cli_foreman_kubevirt/rubygem-hammer_cli_foreman_kubevirt.spec

```diff
@@ -0,0 +1,98 @@
+# FIXME:
+#   1. Edit foreman requirement(s) and specify minimum version
+#   2. Delete these lines
+#
+# Generated from hammer_cli_foreman_kubevirt-0.0.1.gem by gem2rpm -*- rpm-spec -*-
+# template: foreman_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name hammer_cli_foreman_kubevirt
+%global plugin_name hammer_cli_foreman_kubevirt
+%global foreman_min_version FIXME
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?foremandist}%{?dist}
+Summary: Foreman kubevirt commands for Hammer CLI
+Group: Applications/Systems
+License: FIXME
```

Comment:
This indicates that the gem metadata lacks a license. I also don't see a homepage which is also derived from gem metadata. I see these were already added in git. They're part of the v0.1.0 tag but this PR is for 0.0.1. Did you forget to publish 0.1.0?